File: 0701_0800/0711/0711.py
with open("input.txt", "r") as f, open("output.txt", "w") as q:
    n, m = (int(x) for x in f.readline().split())
    d = 1000 * m + 1
    for _ in range(n):
        t = f.readline().strip()
        g = sum(int(x) for x in f.readline().split())
        if g < d:
            d, s = g, t
    q.write(s)

File: 0701_0800/0711/0711.c
#include <stdio.h>
#include <string.h>

int main()
{
	FILE *f = fopen("input.txt", "r"), *q = fopen("output.txt", "w");
	int n, m, d;
	char s[256], t[256];

	fscanf(f, "%d%d", &n, &m);
	d = 1000 * m + 1;
	while (n--)
	{
		int k = 0;
		fscanf(f, "%s", t);
		for (int i = 0, a; i < m; ++i)
			fscanf(f, "%d", &a), k += a;
		if (k < d)
			d = k, strcpy(s, t);
	}
	fprintf(q, "%s", s);
	return 0;
}
